VocaDbWeb/wwwroot/Content/Styles/ExtLinks.less

```
@@ -478,8 +490,13 @@ a.extLink[href^="http://www.yesasia.com/"], a.extLink[href^="https://www.yesasia
 	background: url('/Content/ExtIcons/yesasia.png') no-repeat left 50%;
 }
 
-a.extLink[href^="http://www.youtube.com/"], 
-a.extLink[href^="https://www.youtube.com/"], 
+a.extLink[href^="http://music.youtube.com/"], a.extLink[href^="https://music.youtube.com/"]
+{
+	background: url('/Content/ExtIcons/youtubemusic.png') no-repeat left;

```

```suggestion
	.extLink(url('/Content/ExtIcons/youtubemusic.png'));
```

The same goes for other places. You can replace `background: (url\('([/\w\.]+)'\)) no-repeat left;` with `.extLink($1);` by using regex.